<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>transfermoney</class>
 <widget class="QDialog" name="transfermoney">
  <property name="windowModality">
   <enum>Qt::ApplicationModal</enum>
  </property>
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>513</width>
    <height>131</height>
   </rect>
  </property>
  <property name="minimumSize">
   <size>
    <width>513</width>
    <height>131</height>
   </size>
  </property>
  <property name="maximumSize">
   <size>
    <width>513</width>
    <height>131</height>
   </size>
  </property>
  <property name="windowTitle">
   <string>Transfer</string>
  </property>
  <widget class="QLineEdit" name="senderLineEdit">
   <property name="geometry">
    <rect>
     <x>10</x>
     <y>40</y>
     <width>141</width>
     <height>21</height>
    </rect>
   </property>
   <property name="readOnly">
    <bool>true</bool>
   </property>
   <property name="placeholderText">
    <string>Sender IBAN</string>
   </property>
  </widget>
  <widget class="QLineEdit" name="recvLineEdit">
   <property name="geometry">
    <rect>
     <x>170</x>
     <y>40</y>
     <width>141</width>
     <height>21</height>
    </rect>
   </property>
   <property name="maxLength">
    <number>18</number>
   </property>
   <property name="placeholderText">
    <string>Ex. FI4512345678123411</string>
   </property>
  </widget>
  <widget class="QPushButton" name="execTransaction">
   <property name="geometry">
    <rect>
     <x>420</x>
     <y>40</y>
     <width>80</width>
     <height>21</height>
    </rect>
   </property>
   <property name="text">
    <string>Transfer</string>
   </property>
  </widget>
  <widget class="QLabel" name="transactionStatusLabel">
   <property name="geometry">
    <rect>
     <x>10</x>
     <y>70</y>
     <width>491</width>
     <height>51</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <pointsize>20</pointsize>
    </font>
   </property>
   <property name="frameShape">
    <enum>QFrame::Box</enum>
   </property>
   <property name="text">
    <string/>
   </property>
   <property name="alignment">
    <set>Qt::AlignCenter</set>
   </property>
  </widget>
  <widget class="QLabel" name="label">
   <property name="geometry">
    <rect>
     <x>10</x>
     <y>20</y>
     <width>141</width>
     <height>16</height>
    </rect>
   </property>
   <property name="text">
    <string>Your IBAN</string>
   </property>
  </widget>
  <widget class="QLabel" name="label_2">
   <property name="geometry">
    <rect>
     <x>170</x>
     <y>20</y>
     <width>141</width>
     <height>16</height>
    </rect>
   </property>
   <property name="text">
    <string>Receiver IBAN</string>
   </property>
  </widget>
  <widget class="QPushButton" name="closeButton">
   <property name="geometry">
    <rect>
     <x>420</x>
     <y>10</y>
     <width>80</width>
     <height>21</height>
    </rect>
   </property>
   <property name="text">
    <string>Close</string>
   </property>
  </widget>
  <widget class="QLineEdit" name="amountLineEdit">
   <property name="geometry">
    <rect>
     <x>320</x>
     <y>40</y>
     <width>91</width>
     <height>21</height>
    </rect>
   </property>
   <property name="maxLength">
    <number>9999</number>
   </property>
   <property name="placeholderText">
    <string>Amount</string>
   </property>
  </widget>
 </widget>
 <resources/>
 <connections/>
</ui>
